src/main/java/org/sagebionetworks/web/client/presenter/ErrorPresenter.java

```diff
@@ -55,6 +55,9 @@ public void onSuccess(LogEntry result) {
 				jsniUtils.consoleError(result.getLabel());
 				jsniUtils.consoleError(result.getMessage());
 				jsniUtils.consoleError(result.getStacktrace());
+				if (!synAlert.isUserLoggedIn()) {
+					synAlert.showMustLogin();
```

Comment:
Don't indicate that logging in will gain access to the resource.  Instead, suggest that logging in may help resolve this error (no promises).  "If you feel this error is due to anonymous browsing, please log in".